sys/arm7/main_core/source/main.c

```diff
@@ -22,13 +52,17 @@ void power_button_callback(void)
 void vblank_handler(void)
 {
     inputGetAndSend();
+#if defined(USE_DSWIFI)
     Wifi_Update();
+#endif
 }
 
-int main(int argc, char *argv[])
+int main(void)
 {
     // Initialize sound hardware
+#if defined(USE_MAXMOD) || defined(USE_LIBXM7)
     enableSound();
```

Comment:
Actually, this needs to be left even without any audio library. This is required because libnds has some audio functions that don't depend on external libraries.